build_ovtf.py

```diff
@@ -419,6 +419,8 @@ def main():
                 # ABI 1 TF required latest numpy
                 command_executor(
                     ["pip", "install", "--force-reinstall", "-U numpy"])
+                command_executor(
+                    ["pip", "install", "--force-reinstall", "protobuf==3.20.1"])
```

Comment:
Looks like this is a repeat step as Protobuf is getting installed in the next step too. However this is not impacting anything negatively.